openstack_plugin/resources/compute/host_aggregate.py

```diff
@@ -152,13 +157,17 @@ def delete(openstack_resource):
     Delete host aggregate resource
     :param openstack_resource: Instance of openstack host aggregate resource.
     """
+    if not ctx.instance.runtime_properties.get(RESOURCE_ID):
+        ctx.logger.info('HostAggregate is already uninitialized.')
+        return
```

Comment:
@0lvin Check my previous comment about this

Reply:
will move to decorator.